flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/restore/RocksDBIncrementalRestoreOperation.java

```diff
@@ -653,10 +650,10 @@ private void initBaseDBFromColumnFamilyImports(
                 keyGroupRange,
                 operatorIdentifier);
         rocksHandle.openDB();
-        for (Map.Entry<RegisteredStateMetaInfoBase.Key, List<ExportImportFilesMetaData>> entry :
+        for (Map.Entry<RegisteredStateMetaInfoBase, List<ExportImportFilesMetaData>> entry :
```

Comment:
@rkhachatryan this line looks like it's undoing a change that you introduced when porting ClosableRegistry changes. In the long term, we still want that version because the `RegisteredStateMetaInfoBase.Key` comes from a fix that I committed. So I wonder what's the best order and strategy? Should we stop doing this uncommenting and rather create a branch that first applies the remaining bugfixes and THEN uncomments, or do we start by uncommenting even though it potentially undoes changes like this that we need to redo again. I almost think the first approach is better, wdyt?

Reply:
Discussed offline with @rkhachatryan and we are ok with proceeding with this PR first and then we'll just re-apply all required code for the bugfixes.